### PR TITLE
refactor: Added Nivo :D

### LINKED ISSUE #168: [Fix] Stats at a Glance text overlap
# Overview

Currently, Stats at a Glance text overlaps in the charts.

# Proposal

Format text diagonally to prevent overlap and increase legibility

# Supplemental Information

Current Implementation:
Screen Shot 2022-09-12 at 6 48 22 PM

Ideal Implementation:
Screen Shot 2022-09-12 at 6 49 04 PM

### PR DESCRIPTION
**Pull request by DedsecKnight**
Switched bar chart to use a new framework instead of devexpress. Hopefully resolves #168

### CODE REVIEW

**Review by abdullah50053 — APPROVED** [review]

**Comment by abdullah50053** [comment]
Might want to move everything to Nivo (including pie charts) and remove devexpress altogether

**Comment by adityarathod** [comment]
graphs 😳

**Review by abdullah50053 — APPROVED** [review]

**Comment by abdullah50053** [comment]
@NickBurnett pls review

**Review by abdullah50053 — DISMISSED** [review]

File: components/NivoBarChart.tsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
+import { ResponsiveBar } from '@nivo/bar';
+
+interface NivoBarChartProps {
+  name: string;
+  items: Array<
+    {
+      itemName: string;
+    } & any
```
Comment (NickBurnett) [inline components/NivoBarChart.tsx:8]: what is the point of this? shouldn't it just be any?

**Review by NickBurnett — CHANGES_REQUESTED** [review]

**Comment by abdullah50053** [comment]
@DedsecKnight

**Review by NickBurnett — DISMISSED** [review]

**Comment by NickBurnett** [comment]
@DedsecKnight Please view my comment before merging.

File: components/NivoBarChart.tsx (hunk 1 above)
Comment (DedsecKnight) [reply components/NivoBarChart.tsx:8]: It was an issue with the fact that one attribute name can only determined at runtime. Should be addressed in the next commit.

**Review by abdullah50053 — APPROVED** [review]

**Review by NickBurnett — APPROVED** [review]

**Pull request merged**